src/modules/metaverse/index.ts

```diff
@@ -0,0 +1,66 @@
+/*
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+import { doAPIGet, cleanMetaverseUrl } from "@Modules/metaverse/metaverseOps";
+import { MetaverseInfoResp, MetaverseInfoAPI } from "@Modules/metaverse/APIAccount";
+
+import { Store, Mutations as StoreMutations } from "@Base/store";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { Config } from "@Base/config";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import Log from "@Modules/debugging/log";
+
+// ESLint thinks there are race conditions which don't exist (:fingers-crossed:)
+/* eslint-disable require-atomic-updates */
+
+export const Metaverse = {
+    metaverseUrl: "https://metaverse.vircadia.com/live",
```

Comment:
This URL should be defined as another const of "DEFAULT_METAVERSE_URL", the can then be broken out later.

Reply:
Yes. This string definition will be moved into Config as a constant.